go.mod

```diff
@@ -9,7 +9,6 @@ require (
 	github.com/AdaLogics/go-fuzz-headers v0.0.0-20230811130428-ced1acdcaa24
 	github.com/Masterminds/semver v1.5.0
 	github.com/alecthomas/kong v0.8.1
-	github.com/bufbuild/buf v1.27.2
```

Comment:
What causes `buf` to be installed in the first place without this line? Does `go run` automatically grab it?

Reply:
FWIW this PR sent me on a week long spirit quest that involved refactoring a whole bunch of stuff so we could update to the latest Kubernetes libraries (which in turn breaks the XR controller engine etc) and yet after all that I think this is the best way to address this for the time being. 🙃